cmd/peer-rest-client.go

```diff
@@ -107,6 +108,33 @@ func (client *peerRESTClient) Close() error {
 // GetLocksResp stores various info from the client for each lock that is requested.
 type GetLocksResp map[string][]lockRequesterInfo
 
+// NetReadPerfInfo - fetch network read performance information for a remote node.
+func (client *peerRESTClient) NetReadPerfInfo() (info ServerNetReadPerfInfo, err error) {
+	respBody, err := client.call(
+		peerRESTMethodNetReadPerfInfo,
+		nil,
+		rand.New(rand.NewSource(time.Now().UnixNano())),
+		100*1024*1024, // 100 MiB
```

Comment:
can we make this configurable @balamurugana have provisions for it I here in this API such that we can implement it in admin API as needed.. ?

Reply:
Done